go/tests.yml

```
@@ -80,7 +80,7 @@ jobs:
 {%- endif %}
       with:
```

```suggestion
run: |
    if [ "$(gofmt -l . | wc -l)" -gt 0 ]; then
      echo "Formatting issues found:"
      gofmt -d .
      exit 1
    fi
  timeout-minutes: 30
 ```
 Maybe something around this.

---

Thanks for reviewing, Renata! I will use this code in the dependabot PR to test it.